homework3/semi-uber/Server/api/handlers/trucksHandlers/trucksPatchHandler.js

```diff
@@ -0,0 +1,33 @@
+const log4js = require("log4js");
+const logger = log4js.getLogger();
+const errorHandler = require("../error.handler");
+const { Truck } = require("../../Schemas/truck.schema");
+
+const trucksPatchHandler = async (req, res) => {
+  const { userID, truckID } = req.body;
+
+  try {
+    const clearAssign = await Truck.updateMany(
```

Comment:
Why do you need this variable?
if you just need to call update many without saving the value it returns, you can just call
`await Truck.updateMany(...`

Reply:
Done